hw-js/js/app.js

```diff
@@ -0,0 +1,79 @@
+
+// interpolation- Using a variable to represent a value.
+
+var num = 20
+
+console.log(5 + num);
+
+// concatenation- to combine; put 2 strings together
+
+const pet = "dog";
+const petName = "Eddie";
+
+console.log("I " + " have a " + pet + " and his name is " + petName + "." );
+
+
+// DRY- Don't repeat yourself. We use loops for this, and is an easier way to to repeat.
+
+//BOOLEAN
+
+const a = 4;
+const b = 53;
+const c = 57;
+const d = 16;
+const e = 'Kevin';
+
+console.log(a < b); 
+console.log(c > d); 
+console.log('Name' === 'Name');
+console.log(a < b && b < c); 
+console.log(a === a && a < d); 
```

Comment:
again, you weren't supposed to edit the question...
so you'd want something like: `a * a === d`
you did a good job making them true though :)